server/app/controllers/api/v1/syncs_controller.rb

```diff
@@ -83,6 +85,20 @@ def set_sync
         @sync = current_workspace.syncs.find(params[:id])
       end
 
+      def modify_sync_params
```

Comment:
Why do we have to do this ?

Reply:
It clears out irrelevant parameters before saving them to the database